.github/workflows/test.yml

```
@@ -0,0 +1,76 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test OpenUpgrade migration
+
+on:
+  push:
+    branches: ["14.0*"]
```

Maybe the problem in this?
```suggestion
    branches: ["14.0"]
```